src/core/on-chain/index.ts

```diff
@@ -199,19 +205,40 @@ export const OnChainMixin = (superclass) =>
             sendAll,
           }
 
-          await lockExtendOrThrow({ lock, logger: onchainLoggerOnUs }, async () => {
-            const tx = await ledger.addOnUsPayment({
-              description: "",
-              sats,
-              metadata,
-              payerUser: this.user,
-              payeeUser,
-              memoPayer: memo,
-              shareMemoWithPayee: false,
-              lastPrice: UserWallet.lastPrice,
-            })
-            return tx
-          })
+          const price = await PriceService().getCurrentPrice()
+          if (price instanceof Error) throw price
+          const onChainFee = toSats(0)
+          const usd = sats * price
+          const usdFee = onChainFee * price
```

Comment:
always 0

Reply:
Yup, but I was thinking to leave it in to signal that it's dependent on `onChainFee` & `price` (in case we change the `onChainFee` value to be non-zero).

We can also toy with abstracting this differently maybe?